integrated_channels/integrated_channel/course_metadata.py

```diff
@@ -37,29 +36,24 @@ def get_complete_course_run_details(course_details, course_run_details):
     return course_run_details
 
 
-def get_course_runs(user, enterprise_customer):
+def get_course_runs(enterprise_customer):
     """
     List the course runs the given enterprise customer has in its catalog.
 
-    Args:
-        user: A Django user requesting the course list
+    Arguments:
         enterprise_customer: The given Enterprise Customer
 
     Returns:
         iterable: An iterable containing the details of each course run.
     """
-    catalog_id = enterprise_customer.catalog
-
-    client = CourseCatalogApiClient(user)
+    client = EnterpriseApiClient()
 
-    catalog_courses = client.get_catalog_courses(catalog_id)
-    LOGGER.info('Retrieving course list for catalog %s', catalog_id)
+    enterprise_courses = client.get_enterprise_courses(enterprise_customer, traverse=True).get('results', [])
+    LOGGER.info('Retrieving course list for enterprise %s', enterprise_customer.uuid)
```

Comment:
It might be more readable to put the enterprise customer name instead of the uuid.